plugins/woocommerce/includes/class-wc-brands.php

```diff
@@ -1093,6 +1096,64 @@ public function wc_brands_on_block_template_register( $template_id, $template_ar
 			}
 		}
 	}
+
+	/**
+	 * Hooks the product brand terms block into single product templates.
+	 */
+	public function hook_product_brand_block( $hooked_block_types, $relative_position, $anchor_block_type, $context ) {
+
+		// Only add the block as the last child of the product meta block
+		if ( 'woocommerce/product-meta' === $anchor_block_type &&
+			 'last_child' === $relative_position &&
+			 $context instanceof WP_Block_Template &&
+			 'single-product' === $context->slug ) {
+
+				remove_action( 'woocommerce_product_meta_end', array( $this, 'show_brand' ) );
+
+				// Check if the template already has a product brand block
+				if ( ! $this->template_already_has_brand_block( $context ) ) {
```

Comment:
@Aljullu 

One issue we will have is with stores that:

> Have removed the PHP hook to hide the Brands that were displayed in WC 9.7.
> Have edited the Single Product template to add the Brands taxonomy block:
> I can't think of any way to avoid that, but wanted to highlight it just in case.

I added this bit to cover the case you described above. If a user already has the brand terms in the block, I leave it alone.